src/checkbox/checkbox.js

```diff
@@ -15,255 +15,148 @@
  * limitations under the License.
  */
 
-(function() {
-  'use strict';
+// Numeric constants used in this component.
+const numbers_ = Object.freeze({
+  TINY_TIMEOUT: 0.001
+});
+
+// CSS classes used in this component.
+const classes_ = Object.freeze({
```

Comment:
When trying to implement your suggestion, I ran into some conceptual issues.

I moved the constants inside the constructor:
```js
// Numeric constants used in this component.
    this.numbers_ = Object.freeze({
      TINY_TIMEOUT: 0.001
    });
```

but that leaves them only accessible from an instance of the class. This means that I can neither have the DOM loop outside the class declaration (as in the current code), nor create a static method on the class to handle it either.

As such, I opted to make the constants into static methods on the class:

```js
// Numeric constants used in this component.
  static get numbers_() {
    return {
      TINY_TIMEOUT: 0.001
    };
  }
```

This allows us to access them from static contexts. I turned the component initialization into a static public method to match:

```js
static initComponents(docRoot) {
    var nodes = docRoot.querySelectorAll(
[...]
```